ex1B.py

```diff
@@ -0,0 +1,15 @@
+def numdivision(numlist , num ):
+    numinlist  = len(numlist)
+    counter = 0
+    for numbers in range(numinlist ) :
+        if numlist[counter] % num == 0 :
+            for extranum in range(numinlist - numbers - 1):
```

Comment:
this code is complicated!!!! you have another whole loop only to remove an item from a list please use  list.remove() method instead